### ISSUE TITLE
Xiphos 4.1.0 64bit version

### ISSUE THREAD

**Issue by SuperJAdmin** [crosswire/xiphos#907]
I've just recently put in this version and have noticed that when I click on a verse, the window scrolls automatically to the beginning of the chapter.  It still highlights the verse though as I scroll back down to look at the verse.

**Comment by practicafz** [comment]
I'm having the same problem, same Windows version, same Xiphos version. It sounds a bit like the old XULRUNNER bug from many years ago, that only showed up on Linux.

**Comment by practicafz** [comment]
I tried a previous version of 64-bit Xiphos...same problem. I then tried the 32-bit version of 4.10 and it works fine! So it's an issue with the 64-bit build, or one of it's dependencies.

**Comment by A3M0N** [comment]
I just wanted to mention I'm having the same issue.

**Comment by practicafz** [comment]
Did you try installing the 32-bit version instead?

**Comment by A3M0N** [comment]
No, I didn't. Is there any real benefit to using the 64 bit over 32? If not, I'll switch and try it.

**Comment by practicafz** [comment]
64-bit programs can handle larger chunks of memory, but I'm not sure you would ever run into that with a Bible reading program. I've been using 32-bit since this problem showed up last year, without any difficulty.

**Comment by LAfricain** [comment]
@SuperJAdmin can you try the new release?

**Comment by karlkleinpaste** [comment]
This is apparently due to the implementation of WebKit 1 that's in (only) the Windows build. WK improvements for Windows are a dead letter, and there's nothing we can do about it. I'm not at all clear on how the 32- vs 64-bit version would matter, the behavior should be the same, unless there is a latent bug caused by that architecture difference in just this one area.

**Comment by LAfricain** [comment]
It would be better to change the title of this issue, because I just installed and tried, and I have the same strange behaviour!!!
